include/clink/linalg/sparse_vector.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SPARSE_VECTOR_H_
+#define SPARSE_VECTOR_H_
+
+namespace clink {
+
+// A sparse vector of double values.
+typedef struct SparseVector {
+  SparseVector(const int, const int *, const int, const double *, const int);
```

Comment:
Could we put parameter name in the declaration so that the declaration is more self-explanatory?

Reply:
OK. I'll make the change.